frontend/src/app/main/data/style_dictionary.cljs

```diff
@@ -280,40 +280,44 @@
     (when name-error?
       (wte/error-ex-info :error.import/invalid-token-name (:value schema-error) err))))
 
+(defn parse-json [data]
+  (try
+    (t/decode-str data)
+    (catch js/Error e
+      (throw (wte/error-ex-info :error.import/json-parse-error data e)))))
+
+(defn decode-json-data [data file-name]
```

Comment:
This functions should be private if they are only used on this namespace

Reply:
This function is used on token modal import file